config.example.storage.wikimedia.yaml

```diff
@@ -0,0 +1,113 @@
+                          options:
+                            grace_ttl: 1
+                            index_ttl: 864000
+                            renew_expiring: true
+                            delete_probability: 1
+                            table_name_prefix: parsoid_ng
+                            main_content_type:
+                              name: html
+                              value_type: blob
+                            dependent_content_types:
+                              - name: data-parsoid
+                                value_type: json
+                    /key_value:
+                      x-modules:
+                        - path: sys/key_value.js
+                    /key_rev_value:
+                      x-modules:
+                        - path: sys/key_rev_value.js
+          /{api:sys}:
+            x-modules:
+              - spec:
+                  paths:
+                    /table: &sys_table
+                      x-modules:
+                        - path: sys/table.js
+                          options:
+                            conf:
+                              backend: '{env(RB_TEST_BACKEND, sqlite)}'
+                              hosts: [localhost]
+                              keyspace: system
+                              username: cassandra
+                              password: cassandra
+                              defaultConsistency: one # or 'localQuorum' for production
+                              storage_groups:
+                                - name: default.group.local
+                                  domains: /./
+                              dbname: test.db.sqlite3 # ignored in cassandra, but useful in SQLite testing
+
+
+# The root of the spec tree. Domains tend to share specs by referencing them
+# using YAML references.
+spec_root: &spec_root
+  title: "The RESTBase Storage Service"
+  x-request-filters:
+    - path: lib/security_response_header_filter.js
+  paths:
+    /{domain}: *default_project
+    # A robots.txt to make sure that the content isn't indexed.
+    /robots.txt:
+      get:
+        x-request-handler:
+          - static:
+              return:
+                status: 200
+                headers:
+                  content-type: text/plain
+                body: |
+                  User-agent: *
+                  Allow: /*/v1/?doc
+                  Disallow: /
+
+# Finally, a standard service-runner config.
+info:
+  name: restbase
+
+services:
+  - name: restbase
+    module: hyperswitch
+    conf:
+      port: 7232
+      spec: *spec_root
+      salt: secret
+      default_page_size: 125
+      user_agent: RESTBase
+      ui_name: RESTBase
+      ui_url: https://www.mediawiki.org/wiki/RESTBase
+      ui_title: RESTBase docs
+
+logging:
+  name: restbase
+  level: error
+  streams:
+    - type: stdout
+
+#metrics:
+#  type: statsd
+#  host: localhost
+#  port: 8125
+#  batch: true
+
+ratelimiter:
```

Comment:
Would we need rate limiting for storage given that it's not going to be exposed?

Reply:
indeed we don't

Reply:
Still not needed :P